no-docker-requirements.txt

```diff
@@ -0,0 +1,56 @@
+#########################################################
+#           Armory No-Docker Requirements
+#
+# In order to run armory purely in a python virtual
+# environment, you will need to setup a python environment
+# where python >= 3.7, <3.9.  Additionally the following
+# dependencies are required.  Note: to make this run with
+# pip, users will need to run:
+#         `pip install --upgrade pip==21.0.3`
```

Comment:
This is version `22.0.3` in `docs/no_docker_mode.md`, and I think should be changed to that version.